src/rewards-claimer/LlamaRewardsClaimGuard.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {ILlamaActionGuard} from "src/interfaces/ILlamaActionGuard.sol";
+import {ActionInfo} from "src/lib/Structs.sol";
+import {LlamaRewardsClaimAccountExtension} from "src/rewards-claimer/LlamaRewardsClaimAccountExtension.sol";
+
+/// @title Llama Rewards Claim Guard
+/// @author Llama (devsdosomething@llama.xyz)
+/// @notice A guard that only allows the `RewardsClaimer.claimRewards` to be delegate-called
```

Comment:
We can update this description

Reply:
Done in [5eb0198](https://github.com/llamaxyz/llama-periphery/pull/100/commits/5eb0198dfe436efaf6157161c21f860d8307ffa8)